<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditAttributeDialog</class>
 <widget class="QDialog" name="EditAttributeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>313</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Attribute</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameEditor"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Description:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="descriptionEditor"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="typeCombo">
       <item>
        <property name="text">
         <string>Text</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Generic Number</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Resistance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Inductance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Power</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Percentage</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditAttributeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditAttributeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>258</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>242</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
